raw_to_color_image/app/raw-to-color-node.cc

```diff
@@ -0,0 +1,28 @@
+#include <atomic>
+#include <memory>
+#include <thread>
+
+#include <gflags/gflags.h>
+#include <glog/logging.h>
+#include <ros/ros.h>
+
+#include "raw_to_color/r2c-manager.h"
+#include "raw_to_color/ros-utils.h"
+
+int main(int argc, char** argv) {
+  google::InitGoogleLogging(argv[0]);
+  google::ParseCommandLineFlags(&argc, &argv, true);
+  google::InstallFailureSignalHandler();
+
+  ros::init(argc, argv, "r2c_manager_node");
+  ros::NodeHandle nh, nh_private("~");
+
+  r2c::parseGflagsFromRosParams(argv[0], nh_private);
+  r2c::R2CManager r2c_manager(nh, nh_private);
```

Comment:
nit: I'm not super convinced by the name of the package and this abbreviation. First, "raw" is a term ROS tends to use for just the picture in general how it comes out of the camera (e.g. ros topic suffix image_raw). But in this case we focus on bayer images and only those, so maybe the name could be a bit more specific in that direction. Second, the abbreviation is quite obscure, wouldn't you think so?

Reply:
I couldn't think about something better at the moment. changed the node name now to `image_debayer`